HackYourFuture-CPH/kp-toolbox-fp-class21
# Feature/fp21 46 grids area sorting

# Description
Created logic for sorting tools  component by alphabet and date on frontend,  and added style with responsiveness.

# How to test?
To test by recently added - change in a table "created_at" date for some tools to see the result.
Can be tested in storybook and in the browser.
# Checklist
